docs/howto/openstack/neutron/new-network-control-panel.md

```diff
@@ -1,57 +1,250 @@
-# Creating a new network from the Cleura Cloud Control Panel
+# Creating new networks on the Cleura Cloud
```

Comment:
I suggest to replace "on the Cleura Cloud" with "in Cleura Cloud", which is also what the rest of the page uses.